web-server/v0.4/utils/datastore.js

```diff
@@ -0,0 +1,6 @@
+export default { 
+    run_indices: ".run.*",
+    prefix: "dsa.pbench",
+    elasticsearch: "//elasticsearch.example.com:9280/",
+    pbench_server: "//pbench.example.com"
```

Comment:
Great, but I think you still have references to `production` in the code instead of to `pbench_server`.